tests/EmitTest.elm

```diff
@@ -45,6 +45,14 @@ javascript =
                     , ( "negative int", Literal (Int -998), "-998" )
                     ]
                 )
+            , describe "Float"
+                (List.map runTest
+                    [ ( "positive float", Literal (Float 12.3), "12.3" )
+                    , ( "negative float", Literal (Float -12.3), "-12.3" )
+                    , ( "positive zero float", Literal (Float 0.0), "0" )
+                    , ( "negative zero float", Literal (Float -0.0), "0" )
```

Comment:
Thanks! Can we add this Ellie as a comment? I think it shows some of the behaviour pretty nicely. Eg.
* the positive and negative zeros are equal to each other
* but are different entities - when you divide by them, they result in positive or negative Infinity

Interesting to see that String.fromFloat shows them as the same thing!